retrofit/src/main/java/com/xmartlabs/base/retrofit/module/RestServiceModule.java

```diff
@@ -57,6 +57,6 @@ ServiceStringConverter provideStringConverter() {
   @Provides
   @Singleton
   protected HttpUrl provideBaseUrl(Context context) {
-    return HttpUrl.parse(context.getString(R.string.url_service));
+    return HttpUrl.parse("");
```

Comment:
I think it's a good way of forcing a //TODO. Which in this case every project using this module should do. So, i agree with @matir91